Subsystems/Electronics/Testing/ElectronicLoad/Software/src/main.cpp

```diff
@@ -1,7 +1,76 @@
 #include <mbed.h>
 
+#define HIGH_SHORT (0.0008)
+#define HIGH_ON (0.5)
+#define MED_SHORT (0.0008)
+#define MED_ON (0.05)
+#define LOW_ON (0.005)
+#define GAIN (48 / 3.3)
 
-int main(void) {
+/* Dates:
```

Comment:
Don't need the dates, git commits are timestamped. And already with only one person, this is out of date :P